src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -43,6 +44,17 @@ object DatabaseConversions extends LazyLogging {
     case _ => None
   }
 
+  def extractDateTime(timestamp: java.sql.Timestamp): (Int, Int, Int, String) = {
```

Comment:
Not sure why, but I feel like this will blow up in our face later on?

How do we know that the Timestamp was originally from UTC zone?

Reply:
because Timestamp is universal and it does not have a zone

Reply:
That's my point. I wouldn't be able to guarantee that the time part was actually computed with UTC as reference, that bit of information was lost.

I think that noting this on a doc comment could be fine to start with. It might mention that this function needs to be called with that convention being observed?

Reply:
ok, added the comment